src/Microsoft.VisualStudio.SlowCheetah/Build/Microsoft.VisualStudio.SlowCheetah.targets

```diff
@@ -58,18 +66,41 @@ Copyright (C) Sayed Ibrahim Hashimi, Chuck England 2011-2013. All rights reserve
     <Message Text="SlowCheetah: Collecting Transform files" Importance="low"/>
 
     <ItemGroup>
-      <ScFilesToTransform Include="@(None);@(Content);@(Resource);@(EmbeddedResource)"
-                         Condition=" '%(TransformOnBuild)' == 'true' ">
+      <_ScNoneFilesToTransform Include="@(_NoneWithTargetPath)" Condition="'%(TransformOnBuild)' == 'true'">
+        <BuildAction>None</BuildAction>
```

Comment:
Since the item group is derived from another, this inherit its metadata. Small chance `BuildAction` is already used. Should underscore it to make sure it does not conflict. `_BuildAction`